### ISSUE TITLE
Could not follow _from/_to links in the web interface

### ISSUE THREAD

**Issue by fmarmol** [arangodb/arangodb#5271]
## my environment running ArangoDB
I'm using the latest ArangoDB of the respective release series:
- [ ] 2.8
- [ ] 3.0
- [ ] 3.1
- [ ] 3.2
- [ x] 3.3
- [ ] self-compiled devel branch

Mode:
- [ ] Cluster
- [ x] Single-Server

Storage-Engine:
- [ ] mmfiles
- [x ] rocksdb

On this operating system:
- [ ] DCOS on
  - [ ] AWS
  - [ ] Azure
  - [ ] own infrastructure
- [ ] Linux 
  - [ ] Debian .deb
  - [ ] Ubuntu .deb
  - [ ] SUSE   .rpm
  - [ ] RedHat .rpm
  - [ ] Fedora .rpm
  - [ ] Gentoo
  - [ x] docker - official docker library
  - [ ] other:
- [ ] Windows, version: 
- [ ] MacOS, version:

### this is a web interface-related issue:
I'm using the web interface with this browser: chrome Version 66.0.3359.139   running on this OS: mac Os Sierra v10.13.4
- [ ] authentication is enabled?
- [ ] using the cluster?
- [ ] _system database?

In a document corresponding to an edge, if you click on _to or _from links which contains special character ":", the web interface cannot retrieve the document.

For example:

The document to retrieve has the _id: test/test-1:test-2:test-3:test-4

```
Document not found. Requested ID was: "test/test-1%3Atest-2%3Atest-3%3Atest-4".
```

But it works with a document _id: test/test-1:test-2:test-3

**Comment by maxkernbach** [comment]
Thanks for reporting, i could reproduce it using 3.3.8 on Ubuntu.

For some reason this error does only occur when the document key contains the character ":" exactly 3 times, a document key with a higher or lower amount than 3 of the character ":" is accessible through _from/_to links.

**Comment by jsteemann** [comment]
Fixed in 3.9.0.

**Comment by maxkernbach** [comment]
The [GA release of ArangoDB 3.9](https://www.arangodb.com/2022/02/introducing-arangodb-3-9-further-powering-graph-at-scale/), which includes a fix for this issue, is available.
